tests/cases/notification-generator-sw.js

```diff
@@ -54,6 +54,9 @@ self.addEventListener('notificationclick', function(event) {
       var message = 'Clicked on "' + notification.title + '"';
       if (event.action)
         message += ' (action "' + event.action + '")';
+      if (event.reply) {
```

Comment:
Should we allow distinguishing between NULL and the empty string? Could do something like:

```javascript
if (event.action) {
  message += ' (action: "' + event.action + '", reply: ';
  message += event.reply === null ? 'null' : '"' + event.reply + '"';
  message += ')';
}
```

Reply:
this does mean we will get "reply: undefined" when experimental-web-platform-features are disabled atm, but maybe that's okay?

Reply:
Yeah, it's explicit :)